kong/clustering.lua

```diff
@@ -33,8 +34,10 @@ local table_insert = table.insert
 local table_remove = table.remove
 local inflate_gzip = utils.inflate_gzip
 local deflate_gzip = utils.deflate_gzip
+local pl_tablex_compare = pl_tablex.compare
 
 
+local KONG_VERSION = require("kong.meta")._VERSION
```

Comment:
Does: `_KONG._VERSION` work? I have seen some code use that and some require "kong.meta", not a big deal.